packages/geofire-common/test/Main.test.ts

```diff
@@ -0,0 +1,9 @@
+import * as chai from 'chai';
+
+const expect = chai.expect;
+
+describe('GeoFire GeoCallbackRegistration Tests:', () => {
```

Comment:
I think this test can probably be removed (or improved) not sure how it got here.

Reply:
It's the only test in there, so probably a placeholder? I've commented it out, to at least leave it as an example of what tests should look like.